client/packages/common/src/intl/locales/en/common.json

```
@@ -199,7 +199,7 @@
   "description.rnr-amc": "The Average Monthly Consumption is calculated using the Adjusted consumption value, using this R&R form and the previous two, if available",
   "description.rnr-approved-quantity": "The quantity approved to be ordered. Populated after the R&R form is finalised and authorized.",
   "description.rnr-consumed": "The quantity of this item distributed during this period (via Outbound Shipments or Prescriptions)",
-  "description.rnr-consumed-adjusted": "Shows the consumption adjusted to account for stock out days. =(Consumed x Days in period)/(Days in period - Stock out duration)",
+  "description.rnr-consumed-adjusted": "Shows the consumption adjusted to account for stock out days. = (Consumed x (Days in period / (Days in period - Stock out duration))",
```

```suggestion
  "description.rnr-consumed-adjusted": "Shows the consumption adjusted to account for stock out days. =Consumed x (Days in period / (Days in period - Stock out duration))",
```

Had 3 opening brackets and 2 closing :)

Also I think the no space after `=` was intentional, to help with formatting..

---

Ah thanks for catching! and this was supposed to be the easy one 😅